components/gameboard/TotalScore.tsx

```diff
@@ -16,7 +17,7 @@ function TotalScore() {
     return (
       <li key={player.name} className={styles.playerListItem}>
         <span className={styles.playerName}>{player.name}</span>{" "}
```

Comment:
```suggestion
        <span className={styles.playerName}>{player.name}</span>{" "}
```
Do you need this? {" "}

Reply:
I think I need it for the whitespace between the spans, but I'll check again. Thanks :)